modules/imgproc/src/imgwarp.cpp

```diff
@@ -4368,13 +4368,13 @@ class RemapInvoker :
                                     _mm_storeu_si128((__m128i*)(XY + x1*2 + 8), iy1);
                                 }
                             }
-                        #endif
-
+                        #else 
                             for( ; x1 < bcols; x1++ )
```

Comment:
It's seems the code processes a tail of a data. Previous loop has +8 increment, so if `bcols` is not divisible by 8 some elements won't be processed.